openstack/limes/templates/secret.yaml

```diff
@@ -3,6 +3,6 @@ kind: Secret
 metadata:
   name: limes-secret
 data:
-  postgres_password: {{ required ".Values.postgresql.postgresPassword" .Values.postgresql.postgresPassword | b64enc }}
-  ccloud_auth_password: {{ required ".Values.limes.passwords.ccloud.auth_password" .Values.limes.passwords.ccloud.auth_password | b64enc }}
-  ccloud_rabbitmq_password: {{ required ".Values.limes.passwords.ccloud.rabbitmq_password" .Values.limes.passwords.ccloud.rabbitmq_password | b64enc }}
+  os_password: {{ required ".Values.limes.passwords.ccloud.os_password" .Values.limes.passwords.ccloud.auth_password | b64enc }}
+  postgres_password: {{ required ".Values.postgresql.ccloud.postgresPassword" .Values.postgresql.postgresPassword | b64enc }}
```

Comment:
Please double-check for copy-paste errors. The error messages do not match the actual value paths next to them, and also `auth_password` does not match what's in `ci/test-values.yaml`.